src/services/request-service.ts

```diff
@@ -99,7 +99,7 @@ export class RequestService {
       family: genesisFields?.family,
       model: genesisFields?.model,
     })
-    logger.debug(`Write requested: ${request.cid}`)
+    logger.imp(`Write requested: ${request.cid}`)
```

Comment:
can we make clearer what "write" we're talking about here?

Maybe something like `"Anchor request received for CID ${request.cid} of streamid ${request.streamId} from origin ${request.origin}"`?

Reply:
yes - my intention is to remove this log later but yes good idea